modules/light-clients/08-wasm/internal/ibcwasm/expected_interfaces.go

```diff
@@ -8,13 +8,20 @@ import (
 var _ WasmEngine = (*wasmvm.VM)(nil)
 
 type WasmEngine interface {
-	// Create will compile the wasm code, and store the resulting pre-compile
+	// StoreCode will compile the wasm code, and store the resulting pre-compile
 	// as well as the original code. Both can be referenced later via checksum
 	// This must be done one time for given code, after which it can be
 	// instatitated many times, and each instance called many times.
 	// It does the same as StoreCodeUnchecked plus the static checks.
 	StoreCode(code wasmvm.WasmCode) (wasmvm.Checksum, error)
 
+	// StoreCodeUnchecked will compile the wasm code, and store the resulting pre-compile
```

Comment:
The godoc for this function is identical to that above. Could we add more clarity to differences between `StoreCode` and `StoreCodeUnchecked`

Reply:
Oh.. whoops! Yeah its not super obvious. Any idea what exactly the static checks are? Maybe we could add a brief description on that.

Reply:
Added a suggestion in another comment